controller/src/settings/ui.rs

```diff
@@ -610,6 +610,21 @@ impl SettingsUI {
                         &mut config.health_bar,
                     );
                 }
+                
```

Comment:
As far as I see, only the head dot type is configurable but not the colour nor the thickness 🤔 
You need to add these properties later on in the colour / style table as well :)